src/index.html

```diff
@@ -6,9 +6,87 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item is-active">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item is-active">
            <a
              class="nav__link is-active"
              href="#apple"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#samsung"
            >
              Samsung
            </a>
          </li>
```
add spaces between sibling multiline blocks